spec/item_repository_spec.rb

```diff
@@ -4,310 +4,166 @@
 require 'bigdecimal'
 require './lib/item'
 require './lib/item_repository'
+require './lib/file_io'
+require './data/mock_data'
 
 describe ItemRepository do
   describe '#initialize' do
     it 'exists' do
-      items = []
-      item_repository = ItemRepository.new(items)
+      allow_any_instance_of(ItemRepository).to receive(:create_items).and_return(MockData.items_as_hash)
+      item_repository = ItemRepository.new('fake.csv')
 
       expect(item_repository).is_a? ItemRepository
```

Comment:
🗣 We need to make some decisions about normalizing spacing in our test files. Thinking that should just be a refactor for someone to do independently at the end of the project.

Reply:
Thanks! I made an issue for refactoring our spacing, particularly in spec files. Good catch

Reply:
It's good to run rubocop -A and then seeing if tests are still passing. It will help with some basic formatting and what not. I typically like seeing spaces between sections of code that are not fully related to each other. Definitely a group thing to discuss 👍